src/js/containers/bulkDownload/BulkDownloadPageContainer.jsx

```diff
@@ -127,6 +127,16 @@ export class BulkDownloadPageContainer extends React.Component {
             }
         }
 
+        if (formState.agencyType) {
+            delete params.filters.agency;
```

Comment:
Instead of deleting this from params can you just not include it in the params up above? Since you are generating the `agencies` filter and adding it to params manually there shouldn't be a need to have it as part of the params object.

This should be true for `agency` and `sub_agency`.